src/main/kotlin/org/rust/cargo/project/CargoProjectDescription.kt

```diff
@@ -102,7 +100,7 @@ class CargoProjectDescription private constructor(
 
     fun isCrateRoot(file: VirtualFile): Boolean = findTargetForCrateRootFile(file) != null
 
-    fun withAdditionalPackages(additionalPackages: Collection<Pair<String, VirtualFile>>): CargoProjectDescription {
+    fun withAdditionalPackages(additionalPackages: Collection<Pair<String, VirtualFile>>, origin: PackageOrigin): CargoProjectDescription {
```

Comment:
There're some other dependencies within `ePDWSLC` that must be moved (or removed). I'll get back to it shortly and try to figure out how to do it properly.